apps/storybook/src/Tile.stories.tsx

```diff
@@ -285,3 +285,75 @@ Folder.args = {
   badge: undefined,
   thumbnail: <SvgFolder />,
 };
+
+export const Status: Story<TileProps> = (props) => {
+  const {
+    name,
+    description,
+    metadata,
+    thumbnail,
+    moreOptions,
+    status,
+    ...rest
+  } = props;
+  return (
+    <Tile
+      name={name}
+      description={description}
+      metadata={metadata}
+      thumbnail={thumbnail}
+      moreOptions={moreOptions}
+      status={status}
+      {...rest}
+    />
+  );
+};
+Status.argTypes = {
+  ...Basic.argTypes,
+  thumbnail: { control: { disable: true } },
+};
```

Comment:
Can make `status` into a radio button list

```suggestion
Status.argTypes = {
  ...Basic.argTypes,
  thumbnail: { control: { disable: true } },
  status: { control: 'radio', options: ['positive', 'warning', 'negative'] },
};
```

Reply:
Added